### PR TITLE
Change name to data prep kit

### PR DESCRIPTION
**Pull request by daw3rd**
## Why are these changes needed?

1. Renamed all references to data-prep-lab[-kfp] to data-prep-toolkit[-kfp] and DPL_ to DPK_ and DPLConfig to DPKConfig.
2. Readme and other doc references to the repo are changed from data-prep-lab to data-prep-kit.  Note they won't work though until the rename of this repo is made to data-prep-kit.
3. Reset the version numbers of the new wheels
4. Bumped the minor version number of the published images.

### CODE REVIEW

File: .make.versions [hunk 1]
```diff
@@ -4,24 +4,24 @@
...
+FILTER_VERSION=0.3.0
+NOOP_VERSION=0.8.0
+RESIZE_VERSION=0.3.0
+LANG_ID_VERSION=0.3.0
+TOKENIZER_VERSION=0.3.0
+MALWARE_VERSION=0.4.0
+PROGLANG_SELECT_VERSION=0.3.0
+CODE_QUALITY_VERSION=0.3.0
+DOC_QUALITY_VERSION=0.3.0
+INGEST_TO_PARQUET_VERSION=0.3.0
```
Comment (blublinsky) [inline .make.versions:25]: Why is this 0.3.0? Should we align with the library version and make it 0.1.0?

**Review by blublinsky — APPROVED** [review]
I would suggest to align versions

**Review by roytman — APPROVED** [review]
LGTM

**Comment by revit13** [comment]
@daw3rd please run `make reconcile-requirements` under `kfp/kfp_ray_components` to update image versions in .yaml files. 
Also, please update `.make.transforms_workflows` `kfp/transform_workflows` (renaming lab with kit) and run `make PYTHON=python3.10 build` to update the versions in the workflows.

Thanks

**Review by revit13 — APPROVED** [review]

**Pull request merged**